modules/data.land/R/fia2ED.R

```diff
@@ -30,6 +30,7 @@ fia.to.psscss <- function(settings, gridres=0.075) {
 
 	## connect to database
 	con <-  db.open(settings$database$bety)
+	on.exit(db.close(con))
```

Comment:
Yeah I only learned about it recently. I thought from some PEcAn code, honestly, but perhaps not. Anyway, doing all db.open calls like this would save PEcAn developers that obnoxious max.connections.reached error during bugfixing, if nothing else.

Also note the need for `add=TRUE` argument if doing more than one on.exit() call in a function...